src/aatoolbox/datasources/iri/iri_forecast_seasonal_precipitation_tercile.py

```diff
@@ -0,0 +1,188 @@
+"""Class to download and load IRI's seasonal forecast.
+
+Data is downloaded from `IRI's maproom
+<https://iridl.ldeo.columbia.edu/maproom/Global/Forecasts/NMME_Seasonal_Forecasts/Precipitation_ELR.html>`_
+
+For now only the tercile precipitation forecast has been
+implemented.
+"""
+
+import logging
+
+import requests
+import xarray as xr
+
+from aatoolbox.datasources.datasource import DataSource
+from aatoolbox.utils.geoboundingbox import GeoBoundingBox
+from aatoolbox.utils.raster import AatRasterMixin  # noqa: F401
```

Comment:
Import the entire module rather than just a specific class, just like we do with rioxarray. This gives access to array and dataset methods using `.aat.method()`.

Reply:
perfect! Just did `import aatoolbox.utils.raster as rt`, is that what you meant?